### PR TITLE
Added emscripten_webgl_get_drawing_buffer_size to html5.h

### PR DESCRIPTION
**Pull request by echeese**
There wasn't a way of getting this info before, so I added it. In most cases, this will return the same width and height as the canvas width/height, but that depends on the implementation.

### CODE REVIEW

File: site/source/docs/api_reference/html5.h.rst [hunk 1]
```diff
@@ -2044,6 +2044,16 @@ Functions
 	:rtype: |EMSCRIPTEN_WEBGL_CONTEXT_HANDLE|
 	
 	
+	
+.. c:function:: void emscripten_webgl_get_drawing_buffer_size(EMSCRIPTEN_WEBGL_CONTEXT_HANDLE context, int *width, int *height)
```
Comment (juj) [inline site/source/docs/api_reference/html5.h.rst]: Can you change the return type to `EMSCRIPTEN_RESULT`?

File: site/source/docs/api_reference/html5.h.rst [hunk 2]
```diff
@@ -2044,6 +2044,16 @@ Functions
...
+
+	Gets the drawingBufferWidth/drawingBufferHeight of the specified WebGL context.
+
+	:param EMSCRIPTEN_WEBGL_CONTEXT_HANDLE context: The WebGL context to get width/height of.
+	:param int* width: The context's drawingBufferWidth.
+	:param int* height: The context's drawingBufferHeight.
```
Comment (juj) [inline site/source/docs/api_reference/html5.h.rst]: The preference is to write `int *width` instead of `int* width`.

File: src/library_html5.js [hunk 3]
```diff
@@ -1865,6 +1865,12 @@ var LibraryJSEvents = {
     return GL.currentContext ? GL.currentContext.handle : 0;
   },
 
+  emscripten_webgl_get_drawing_buffer_size: function(contextHandle, width, height) {
+    var GLContext = GL.getContext(contextHandle).GLctx;
```
Comment (juj) [inline src/library_html5.js]: If `GL.getContext(contextHandle)` returns null, or if `GL.getContext(contextHandle).GLctx` is not a valid context, then this function should return `{{{ cDefine('EMSCRIPTEN_RESULT_INVALID_PARAM') }}}` and not abort execution by throwing.

File: src/library_html5.js [hunk 4]
```diff
@@ -1865,6 +1865,12 @@ var LibraryJSEvents = {
...
+    {{{ makeSetValue('width', '0', 'GLContext.drawingBufferWidth', 'i32') }}};
+    {{{ makeSetValue('height', '0', 'GLContext.drawingBufferHeight', 'i32') }}};
```
Comment (juj) [inline src/library_html5.js]: Likewise, if `width` or `height` pointer is null, this should not write to the pointers, but would be good to return `{{{ cDefine('EMSCRIPTEN_RESULT_INVALID_PARAM') }}}`.

File: src/library_html5.js [hunk 5]
```diff
@@ -1865,6 +1865,12 @@ var LibraryJSEvents = {
...
+  },
```
Comment (juj) [inline src/library_html5.js]: On success, then return `{{{ cDefine('EMSCRIPTEN_RESULT_SUCCESS') }}}`

**Comment by juj** [comment]
Thanks, this looks like a good feature! Can you make sure your name is in AUTHORS, and also rebase to remove the merge commit.

**Comment by echeese** [comment]
Alright, I made those changes. My name's already in the AUTHORS.

**Comment by juj** [comment]
Thanks, very good! I'll merge this in and do some formatting and add a test as well.

**Pull request merged**